doc/modules/clustering.rst

```diff
@@ -799,6 +805,88 @@ by black points below.
    In Proceedings of the 2nd International Conference on Knowledge Discovery
    and Data Mining, Portland, OR, AAAI Press, pp. 226–231. 1996
 
+.. _optics:
+
+OPTICS
+======
+
+The :class:`OPTICS` algorithm shares many similarities with the :class:`DBSCAN`
+algorithm, and can in fact be considered a generalization of DBSCAN that
+relaxes the ``eps`` requirement from a single value to a value range. The key
+difference between DBSCAN and OPTICS is that the OPTICS algorithm builds a
+*reachability* graph, which assigns each sample both a ``_reachability``
+distance, and an spot within the cluster ``ordering_`` attribute; these two
+attributes are assigned when the model is fit, and are used to determine
+cluster membership. If OPTICS is run with the default value of *inf* set for
+``max_bound``, then DBSCAN style cluster extraction can be performed in linear
+time for any given ``eps`` value using the ``extract_dbscan`` method. Setting
+``max_bound`` to a lower value will result in shorter run times, and can be
+thought of as the maximum cluster object size that OPTICS will be able to
```

Comment:
maximum size in what sense? number of samples or diameter?

Reply:
good point-- it's diameter, updated in documentation.